script/Ciel/Day 5/05_01_CIEL05_3C.txt

```diff
@@ -548,17 +549,18 @@ No matter what, I can't let the tragedy of what happened at the hotel repeat aga
 [sha:054c650ad6be9cdd0ef2598f9524d7e5edfb28e8]{
 -- Page 535, Offset 12249.
 -- 「―――分かった。頼むアルクェイド。
-"―――I understand. I ask of you Arcueid.
+// begging might be a bit too strong here
+"―――I understand. I'm begging you, Arcueid.
```

Comment:
next line already starts with please so i say just drop it... as dangerous as that is with voiced lines
>I understand. Arcueid―――